packages/fxa-auth-server/lib/senders/emails/partials/changePassword/en-US.ftl

```diff
@@ -0,0 +1,2 @@
+change-password-plaintext = If you suspect that someone is trying to gain access to your account, please change your password.
+{ $passwordChangeLink }
```

Comment:
I don't think FTL likes this. Could it be inline with the localization?

Reply:
Or actually, if the link is _supposed_ to be on a new line I would recommend moving the rendering of the link to EJS in the plaintext:

```
change-password-plaintext = "If you suspect that someone is trying to gain access to your account, please change your password."
<%- passwordChangeLink %>
```

In this case we wouldn't include it in the FTL at all.